docs/phase1_limitations.md

```diff
@@ -0,0 +1,11 @@
+# Phase 1 Limitations
+
+
+## Limitation
+The default SA in kcp workspace has the secret containing its token synchronised in the workload cluster but under a different name (with kcp prefix). In phase 1 we don't want Tekton pods to communicate with the kcp API. As such they should not use an SA and its token created by kcp. This is a limitation but existing SA from the workload cluster can be used instead.
+
+## Cases Instructions
+* Not specifying an SA is OK
+* pipeline SA can be used
+* default SA does not work and would point to kcp API
+* Custom SA created in kcp workspace may have the same flaw for our phase 1 approach
```

Comment:
=> A custom SA created in kcp workspace will present the same flaw with phase 1.

Reply:
Fixed